src/ServiceBusChannel.php

```diff
@@ -150,13 +150,25 @@ private function getToken(): string
             $this->generateTokenKey(),
             function () {
                 try {
-                    $response = $this->client->request(
-                        'POST',
-                        $this->getUrl('login'),
-                        [
-                            'json' => Arr::only($this->ventureConfig, ['username', 'password', 'venture_config_id']),
-                        ]
-                    );
+                    $version = intval($this->config['version']);
+
+                    if ($version < 2) {
+                        $response = $this->client->request(
+                            'POST',
+                            $this->getUrl('login'),
+                            [
+                                'json' => Arr::only($this->config, ['username', 'password', 'venture_config_id']),
+                            ]
+                        );
+                    } else {
+                        $response = $this->client->request(
+                            'POST',
+                            $this->getUrl('login'),
+                            [
+                                'json' => Arr::only($this->config, ['username', 'password', 'node_id']),
```

Comment:
shouldn't it be `from` instead of `node_id`?

Reply:
@datashaman not for the login request. it's `from` for the events endpoint and `node_id` for the login endpoint